keps/0019-package-api-versioning.md

```diff
@@ -165,6 +165,7 @@ Two proposals: singular and composite package/operator version. In the case of p
 
 - Visually simple
 - It can be known just from the package version whether or not it contains a certain operator-related feature or bug fix
+- Version is a combination of the application and operator in a single SemVer.
```

Comment:
IMO, it is not :). It can be anything SemVer, we're not forcing operator devs to anything here. Though, with things like backports in mind, it makes sense to couple certain application version changes to operator version changed.

Reply:
I think getting a precise phrasing here is tricky, but afaik the intention is to mean that the *version* is tied to an *application version* plus a specific set of operator-features. (@zmalik to confirm 😛)

Reply:
yes, I meant originally from where it's derived